README.md

```diff
@@ -1 +1,30 @@
-# boilerplate-react-native
+# Boilerplate React Native
```

Comment:
I am not sure if the list here is exhaustive.

For example, I see `Boilerplate` in `android/app/src/main/java/com/boilerplate/MainActivity.java`.

Recommend to do a thorough self review, identify all places that uses boilerplate in some fashion and highlight here.

cc @bbbneo333